docs/02.API-REFERENCE.md

```diff
@@ -2021,6 +2036,61 @@ main (void)
 - [jerry_create_typedarray](#jerry_create_typedarray)
 
 
+## jerry_get_container_type
```

Comment:
This doc entry should be placed after the `jerry_value_get_type` method (it is a bit similar to that).
In addition what if we use the name `jerry_value_get_container_type` ?

Reply:
The previous name was `jerry_value_get_container_type` however other API functions use the `jerry_get_...` (`jerry_get_error_type`, `jerry_get_array_length`, `jerry_get_string_size`) scheme without the `value`.